presto-native-execution/presto_cpp/main/operators/tests/UnsaferowShuffleTest.cpp

```diff
@@ -306,15 +332,91 @@ TEST_F(UnsafeRowShuffleTest, endToEnd) {
     params.destination = i;
 
     bool noMoreSplits = false;
-    auto [taskCursor, serializedResults] = readCursor(params, [&](auto* task) {
+    auto [taskCursor, output] = readCursor(params, [&](auto* task) {
       if (noMoreSplits) {
         return;
       }
       addRemoteSplits(task, {leafTaskId});
       noMoreSplits = true;
     });
 
+    ASSERT_EQ(output.size(), 1);
+    auto vector= copyResultVector(output[0]);
```

Comment:
space before '='; wondering why format-check didn't catch this; would you run make format-fix to make sure there are no other formatting errors?